src/app/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import type { ReactNode } from 'react';
+import styles from './ActionButton.module.css';
+
+type ActionButtonProps = {
+  children: ReactNode;
+  onClickHandle?: () => void;
```

Comment:
`onClick` is better and then, once you use it, name the assigned function `handleClick`

Reply:
Done so